### PR TITLE
Add post about GOV.UK Prototype Filters

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
Depends on https://github.com/x-govuk/govuk-prototype-filters/issues/18

Post covers:

* The introduction of plugins with v13 of the Prototype Kit
* What Nunjucks filters are
* Examples of how the filters in this plugin align with guidance in the GOV.UK Style Guide
* Installation instructions
* How to contribute

And yes, I couldn’t help but create a new homepage illustration, I’ll be adding this to the docs for the plugin 😉

### CODE REVIEW

File: app/posts/govuk-prototype-filters.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+description: Introducing a collection of Nunjucks template filters to help manipulate and transform data used in prototypes.
+date: 2023-05-21
+tags: post
+layout: post
+author:
+  name: Paul Lloyd
+  url: https://github.com/paulrobertlloyd
+image:
+  src: /assets/posts/govuk-prototype-filters/govuk-prototype-filters.png
+  alt: The homepage illustration for the GOV.UK Prototype Filters.
```
Comment (frankieroberto) [inline app/posts/govuk-prototype-filters.md:12]: Think the alt text should describe the image? Or else maybe it’d be better for it to be blank (empty string) if it’s purely decorative? 🤔 

```suggestion
  alt: Illustration titled 'GOV.UK Prototype Filters' showing checkboxes and radio buttons being generated from code.
```

**Review by frankieroberto — APPROVED** [review]
Love it! Let a few minor comments.

You could consider making the intro a bit more personal / blog-style (ie using "I"), and perhaps talking about how/when you started this and why? But that's up to you.

Might also be nice to mention/thank anyone who's contributed (either directly or indirectly) so far within the "contribute" section? Partly to be nice, and partly as social-proof to encourage others to contribute! 😄

File: app/posts/govuk-prototype-filters.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  opengraphImage: true
+---
+
+One of the most exciting features included in version 13 of the [GOV.UK Prototype Kit](https://prototype-kit.service.gov.uk/docs/) was its new plugin system.
+
+Plugins allow you to use extra components, styles and other features, either from within your own organisation’s design system or provided by the wider community.
+
+Plugins can also provide additional filters that can be used in Nunjucks templates. If you already use [Nunjucks](https://mozilla.github.io/nunjucks/), you may be familiar with the concept of filters, with several already built in.
+
+For example, if you want to ensure a postcode is always shown using uppercase letters, you can use the [`upper`](https://mozilla.github.io/nunjucks/templating.html#upper) filter:
```
[the remark this answers is not in the document]
Comment (paulrobertlloyd) [reply app/posts/govuk-prototype-filters.md]: Done!

**Comment by paulrobertlloyd** [comment]
Take a look at the latest tweaks. Don’t know if you @frankieroberto and @fofr would like a mention, namely for putting up with my continued stream of PRs? 😉

**Review by frankieroberto — APPROVED** [review]
Love it.

**Pull request merged**